### PR TITLE
CDS-53904 making org and project optional

### PR DESCRIPTION
**Pull request by RajBaviskar**
## Describe your changes

- making org and project optional

## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`
- Sub Category Field Check: `trigger subcategoryfieldcheck`

### CODE REVIEW

File: docs/resources/platform_environment.md [hunk 1]
```diff
@@ -80,14 +80,14 @@ resource "harness_platform_environment" "example" {
...
 - `identifier` (String) Unique identifier of the resource.
 - `name` (String) Name of the resource.
-- `org_id` (String) Unique identifier of the organization.
-- `project_id` (String) Unique identifier of the project.
 - `type` (String) The type of environment. Valid values are PreProduction, Production
 
 ### Optional
 
 - `color` (String) Color of the environment.
 - `description` (String) Description of the resource.
```
Comment (rathodmeetsatish) [inline docs/resources/platform_environment.md:88]: import examples are not reflecting in doc and please add org level and account level examples

File: docs/resources/platform_service.md [hunk 2]
```diff
@@ -75,12 +75,12 @@ resource "harness_platform_service" "example" {
 
 - `identifier` (String) Unique identifier of the resource.
 - `name` (String) Name of the resource.
-- `org_id` (String) Unique identifier of the organization.
-- `project_id` (String) Unique identifier of the project.
```
Comment (rathodmeetsatish) [inline docs/resources/platform_service.md:78]: import examples are not reflecting in doc and please add org level and account level examples

File: docs/data-sources/platform_environment.md [hunk 3]
```diff
@@ -23,15 +23,12 @@ data "harness_platform_environment" "example" {
...
 ## Schema
 
-### Required
-
-- `org_id` (String) Unique identifier of the organization.
-- `project_id` (String) Unique identifier of the project.
-
 ### Optional
 
 - `identifier` (String) Unique identifier of the resource.
```
Comment (rathodmeetsatish) [inline docs/data-sources/platform_environment.md:28]: identifier should be required right?

File: docs/data-sources/platform_service.md [hunk 4]
```diff
@@ -23,15 +23,12 @@ data "harness_platform_service" "example" {
...
 ## Schema
 
-### Required
-
-- `org_id` (String) Unique identifier of the organization.
-- `project_id` (String) Unique identifier of the project.
-
 ### Optional
 
 - `identifier` (String) Unique identifier of the resource.
```
Comment (rathodmeetsatish) [inline docs/data-sources/platform_service.md:28]: identifier should be required right?

File: internal/service/platform/service/resource_service_test.go [hunk 5]
```diff
@@ -87,6 +87,82 @@ func TestAccResourceServiceWithYaml(t *testing.T) {
...
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "id", id),
+					resource.TestCheckResourceAttr(resourceName, "name", name),
+				),
+			},
+			{
+				Config: testAccResourceServiceWithYamlOrgLevel(id, updatedName, updatedVarValue),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "id", id),
+					resource.TestCheckResourceAttr(resourceName, "name", updatedName),
```
Comment (rathodmeetsatish) [inline internal/service/platform/service/resource_service_test.go:153]: can we assert org_id similarly in other places

**Review by rathodmeetsatish — CHANGES_REQUESTED** [review]

File: docs/data-sources/platform_environment.md (hunk 3 above)
Comment (RajBaviskar) [reply docs/data-sources/platform_environment.md:28]: changed.

File: docs/data-sources/platform_service.md (hunk 4 above)
Comment (RajBaviskar) [reply docs/data-sources/platform_service.md:28]: changed.

File: docs/resources/platform_environment.md (hunk 1 above)
Comment (RajBaviskar) [reply docs/resources/platform_environment.md:88]: added.

File: docs/resources/platform_service.md (hunk 2 above)
Comment (RajBaviskar) [reply docs/resources/platform_service.md:78]: added.

File: internal/service/platform/service/resource_service_test.go (hunk 5 above)
Comment (RajBaviskar) [reply internal/service/platform/service/resource_service_test.go:162]: added

**Review by rathodmeetsatish — APPROVED** [review]

**Pull request merged**